Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiManager.h

```diff
@@ -91,5 +92,7 @@ namespace AzToolsFramework::ViewportUi
         void UpdateSwitcherButtonGroupUi(Internal::ButtonGroup* buttonGroup);
         //! Update the corresponding ui element for the given text field.
         void UpdateTextFieldUi(Internal::TextField* textField);
+
+        AZ::u64 m_currentId = 0;
```

Comment:
suggestion: This looks good but can we make the variable name a little clearer, I  guess this is `m_nextViewportUiElementId` technically?